cmake/modules/FindMathJax.cmake

```diff
@@ -0,0 +1,41 @@
+## ---------------------------------------------------------------------
+##
+## Copyright (C) 2013 - 2016 by the deal.II authors
+##
+## This file is part of the deal.II library.
+##
+## The deal.II library is free software; you can use it, redistribute
+## it, and/or modify it under the terms of the GNU Lesser General
+## Public License as published by the Free Software Foundation; either
+## version 2.1 of the License, or (at your option) any later version.
+## The full text of the license can be found in the file LICENSE at
+## the top level of the deal.II distribution.
+##
+## ---------------------------------------------------------------------
+
+#
+# Try to find the MATHJAX library
+#
+# This module exports
+#
+#   MATHJAX_PATH
+#
+
+# SET(MATHJAX_DIR "" CACHE PATH "An optional hint to a MATHJAX installation")
+# SET_IF_EMPTY(MATHJAX_DIR "$ENV{MATHJAX_DIR}")
+
+DEAL_II_FIND_PATH(MATHJAX_PATH MathJax.js
+  PATHS ${MATHJAX_ROOT}
+  $ENV{MATHJAX_ROOT}
+  "${CMAKE_PREFIX_PATH}/share/javascript/mathjax"
+  "${CMAKE_INSTALL_DATADIR}/javascript/mathjax"
+  "/usr/share/javascript/mathjax"
+  "/usr/share/yelp/mathjax"
+  DOC "Root path of MathJax.js"
+  )
```

Comment:
I would prefer this check to be a simple statement in `doc/doxygen/CMakeLists.txt`.

The `CONFIGURE_FEATURE()` is really only meant for direct library dependencies.

Would you mind to remove `configure_1_mathjax.cmake ` and `FindMathJax.cmake` and simply
put the call
```
IF(NOT DEAL_II_DOXYGEN_USE_ONLINE_MATHJAX)
  DEAL_II_FIND_PATH(MATHJAX_PATH MathJax.js
    PATHS ${MATHJAX_ROOT}
    $ENV{MATHJAX_ROOT}
    "${CMAKE_PREFIX_PATH}/share/javascript/mathjax"
    "${CMAKE_INSTALL_DATADIR}/javascript/mathjax"
    "/usr/share/javascript/mathjax"
    "/usr/share/yelp/mathjax"
    DOC "Root path of MathJax.js"
  )
ENDIF()
```
at the appropriate place in `doc/doxygen/CMakeLists.txt `?

Please error out with a `FATAL_ERROR` if it cannot be found.